Library/Formula/ponyc.rb

```diff
@@ -0,0 +1,16 @@
+class Ponyc < Formula
+  homepage "http://www.ponylang.org"
+  url "http://www.ponylang.org/releases/source/ponyc-0.1.1.tar.bz2"
+  version "0.1.1"
+  sha256 "8ed8e5139635cf7f60cf87d51fd36210ab11a8a7fcefbdfa9b5c25a2666bee1e"
+
+  depends_on "homebrew/versions/llvm36"
```

Comment:
With 0536279fd6186775ec514f218484709e2c0f81d9, I have updated the llvm formula to the latest official release, as it was outdated. With this, the ponyc formula does not depend on any non-core formulas.

Submitting a new pull request!